pytorch_lightning/utilities/xla_device.py

```diff
@@ -71,6 +71,10 @@ def _is_device_tpu() -> bool:
         # we would have to use `torch_xla.distributed.xla_dist` for
         # multiple VMs and TPU_CONFIG won't be available, running
         # `xm.get_xla_supported_devices("TPU")` won't be possible.
+
+        # Temporary to support tpu+vm arch
+        if "XRT_TPU_CONFIG" not in os.environ:
+            return False
```

Comment:
won't this change existing behavior for TPU training?

Reply:
Nope, as the env variable is required by PyTorch XLA

Reply:
@kaushikb11 @tchaton n00b question: why does the comment say its temporary then?